vendor/tooltipscan.lua

```diff
@@ -45,6 +89,30 @@ function Addon:ImportTooltipTextRight(tooltip, bag, slot)
     return importTooltipTextToTable(tooltip, "Right", bag, slot)
 end
 
+function Addon:ImportTooltipColorLeft(tooltip, bag, slot)
+    return importTooltipColorToTable(tooltip, "Left", bag, slot)
```

Comment:
The imports are unnecessary since it has no value other than the initial red detection.